dandischema/datacite/__init__.py

```diff
@@ -148,13 +148,19 @@ def to_datacite(
         {"description": meta.description, "descriptionType": "Abstract"}
     ]
 
+    # Populate publisher info
     attributes["publisher"] = {
         "name": f"{instance_config.instance_name} Archive",
-        "schemeUri": "https://scicrunch.org/resolver/",
-        "publisherIdentifier": f"https://scicrunch.org/resolver/{instance_config.instance_identifier}",
-        "publisherIdentifierScheme": "RRID",
         "lang": "en",
     }
+    if instance_config.instance_identifier is not None:
```

Comment:
I do prefer generic programming

```suggestion
    if instance_config.instance_identifier:
```

shorter, simpler code is easier to read/follow since removes questions on eg what should happen if it is empty? Why should it be different?